config/locales/cookies_preferences.yml

```diff
@@ -91,7 +91,7 @@ en:
           We also use additional cookies to measure how you use the service and to help us with our marketing and communications.
 
       heading: Cookies on Teaching Vacancies
-      subheading: 
```

Comment:
`name: consented-to-additional-cookies` in line 81 needs an update of the name too, as it is used for informing the user about the name of the cookie stored cookie in their session.

![image](https://github.com/user-attachments/assets/1a71958d-4cc1-4c92-969a-f031e52ef630)